Add example with MethodViews and Blueprints

nicholasRutherford opened the pull request:
Added an example using MethodViews and Blueprints. See #23

jirikuncar commented on docs/index.rst:
```diff
@@ -203,6 +203,66 @@ with 3 items during processing request for `/social/list`.
                    list(breadcrumbs.current_breadcrumbs)) == [
                       '/', '/account/', '/social/list']
 
+Complicated Examples
```
What about calling it "Advanced Examples"? "Complicated" might scare people ;)

jirikuncar commented:
@nicholasRutherford thanks a lot for your contribution.

- [ ] Please add yourself to `AUTHORS`.
- [ ] Please reply if you agree to donate this (and any future) contributions to our Flask-Breadcrumbs package under the terms of the Revised BSD Licence.

Can you also amend commit message to follow our style?

```
docs: example using MethodViews and Blueprints

* BETTER Adds an advanced example using MethodViews and Blueprints.
  (closes #23)
```

nicholasRutherford commented:
@jirikuncar Good call on the renaming! I've also updated the commit message and added myself to authors (though I feel unworthy). I also agree to donate this (and any future) contributions to the Flask-Breadcrumbs package under the terms of the Revised BSD Licence.
